### ISSUE TITLE
kfilemetadata 5.87.0 has bogus checks for ffmpeg

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/distribution#2749]
kfilemetadata-5.87.0 needs this
```

-- Checking for module 'libavutil>=57.48'
--   Package dependency requirement 'libavutil >= 57.48' could not be satisfied.
Package 'libavutil' has version '56.70.100', required version is '>= 57.48'
-- Checking for module 'libswscale>=57.48'
--   Package dependency requirement 'libswscale >= 57.48' could not be satisfied.
Package 'libswscale' has version '5.9.100', required version is '>= 57.48'
-- Checking for module 'libpostproc>=57.48'
--   Package dependency requirement 'libpostproc >= 57.48' could not be satisfied.
```

**Comment by berolinux** [comment]
We have the latest version of ffmpeg. The problem is something else -- the cmake files for kfilemetadata assume that all parts of ffmpeg are versioned like libavcodec (we have 58.134.100), but in reality, subcomponents have different version numbers.

**Comment by berolinux** [comment]
And this is fixed in kfilemetadata-5.88.0

It now correctly demands

find_package(FFmpeg 57.48 COMPONENTS AVCODEC)
find_package(FFmpeg 57.40 COMPONENTS AVFORMAT)
find_package(FFmpeg 55.27 COMPONENTS AVUTIL)

(we have avcodec 58.134.100, avformat 58.76.100, avutil 56.70.100 - all part of ffmpeg 4.4.1)
